Task 5/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <script src='main.js'></script>
+</head>
+
+<body>
+    <h1>
+        CV
+    </h1>
+    <hr>
+    <h2>
+        Contacts:
+    </h2>
+    <p>
+    <h3>Bashkatov Roman</h3>
+    Phone: <a href="tel:+79777777777">+79777777777</a>
+    Email: <a href="mailto:email@mail.ru">email@mail.ru</a>
+    </p>
+    <h2>
+        About me
+    </h2>
+    Like game, serial and coding
+    <h3>
```

Comment:
Don't leave text in body without wrapping it in something else (like `<p>` or `<span>`)